public/js/profile.web.js

```diff
@@ -5,8 +5,11 @@ MatrixProfile.prototype.loadStoredProfile = function loadStoredProfile() {
 };
 
 MatrixProfile.prototype.userName = function userName() {
-  const user = JSON.parse(localStorage.getItem('user')).name;
-  return user.split(' ').pop();
+  const user = JSON.parse(localStorage.getItem('user'));
+  if(user){
+    return user.name.split(' ').pop();  
+  }
+  return "";
```

Comment:
Maybe return `Anonyms` or maybe we could treat this in the view?

Reply:
Return null?